packages/bitcore-wallet-service/config.js

```diff
@@ -21,47 +21,46 @@ var config = {
 
   storageOpts: {
     mongoDb: {
-      uri: 'mongodb://localhost:27017/bws',
-    },
+      uri: 'mongodb://localhost:27017/bws'
+    }
   },
   messageBrokerOpts: {
     //  To use message broker server, uncomment this:
     messageBrokerServer: {
-      url: 'http://localhost:3380',
-    },
+      url: 'http://localhost:3380'
+    }
   },
   blockchainExplorerOpts: {
     btc: {
       livenet: {
-        url: 'https://api.bitcore.io',
+        url: 'https://api.bitcore.io'
       },
       testnet: {
         url: 'https://api.bitcore.io',
-      },
+        regtestEnabled: false
```

Comment:
For internal testing, change url to localhost:3000 and set regtestEnabled to true